examples/reverse-proxy/src/main/java/example/armeria/reverseproxy/Main.java

```diff
@@ -0,0 +1,85 @@
+package example.armeria.reverseproxy;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.client.grpc.GrpcClients;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.internal.common.util.PortUtil;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.Param;
+import com.linecorp.armeria.server.annotation.ProducesJson;
+import com.linecorp.armeria.server.annotation.decorator.LoggingDecorator;
+import com.linecorp.armeria.server.grpc.GrpcService;
+
+public final class Main {
+
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    private static final int serverPort = PortUtil.unusedTcpPort();
+    private static final int backendServerPort = PortUtil.unusedTcpPort();
```

Comment:
In an example, we should use the fixed ports. I'd expect tests use unused random ports though.

Reply:
I modified it to use a fixed port.

https://github.com/line/armeria/pull/5722/commits/9424ddbc89eddaa53cc7193a6771d9bac5955284